packages/help-center/src/components/help-center-contact-form.tsx

```diff
@@ -289,6 +290,14 @@ export const HelpCenterContactForm = () => {
 						`Plan: ${ productId } - ${ productName } (${ productTerm })`,
 					];
 
+					if ( getQueryArgs()?.ref === 'woocommerce-com' ) {
+						ticketMeta.push(
+							isWcMobileApp()
+								? 'Created during store setup on Woo mobile app'
+								: ' Created during store setup on Woo browser'
```

Comment:
Can we keep the `key: value` format? Maybe we'll parse these some time in the future.

`Created: during store setup on ${isWcMobileApp() ? 'Woo mobile app' : 'browser'}.`

Reply:
Yeah, sounds good! 👍